baseapp_chats/triggers.py

```
@@ -49,13 +49,15 @@ def create_message_status_trigger(ChatRoomParticipant, MessageType):
         operation=pgtrigger.Insert,
         condition=pgtrigger.Q(new__message_type=MessageType.USER_MESSAGE),
-        func=Func(f"""
+        func=Func(
+            f"""
             INSERT INTO {{model.statuses.field.model._meta.db_table}} (message_id, profile_id, is_read)
             SELECT NEW.id, crp.profile_id, CASE WHEN NEW.profile_id = crp.profile_id THEN TRUE ELSE FALSE END
             FROM {ChatRoomParticipant._meta.db_table} as crp
             WHERE crp.room_id = NEW.room_id;
             RETURN NULL;
-        """),
+        """
```

Ruff (0.15.6)

[error] Possible SQL injection vector through string-based query construction

(S608)